app/models/admin_section.rb

```diff
@@ -10,6 +10,10 @@ def self.permissions
     end
   end
 
+  def self.permissions_for_account(account)
+    account.provider_can_use?(:policy_registry) ? permissions : (permissions - %i[policy_registry])
```

Comment:
I think we're missing the corresponding feature to be defined at https://github.com/3scale/porta/blob/master/app/lib/logic/rolling_updates.rb#L52

Reply:
@guicassolato My bad, I meant to write this instead:
```suggestion
    account.provider_can_use?(:policies) ? permissions : (permissions - %i[policies])
```

Reply:
Is it really the same? I mean, doesn't it make sense to enable policies (in general) without yet enabling the (custom) policies registry?

Reply:
@guicassolato I don't know... in this PR I meant to make it `policies` in the whole PR because I thought it was either enabling policies in general or no policies at all, but I could be mistaken and we could need another new rolling_update for the custom policies, but in that case keep it in mind for every reference to rolling update in this PR (the 2 last commits that I left here :+1: ).

Reply:
Decided for a new rolling update `policies_registry` that depends on the already existing `policies` one. So one can activate policies in general but without the endpoint to create a custom policy.